From 537b43e9c3866ab81ba0ab1fe078ae8569f113b0 Mon Sep 17 00:00:00 2001
From: mbrinkma <mbrinkma@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Mon, 23 Mar 2020 21:41:40 +0000
Subject: [PATCH 367/462] MANTIS 0041015 BuildManagement - add build-parse
 rules for Jenkins - swapped maven and java build-order according to
 recommendations

git-svn-id: svn://localhost/SelfServiceBuild/trunk@673 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../Build/jenkins/plf-build--parse-rules.txt    | 17 +++++++++++++++++
 SelfServiceBuild/Build/release/makefile-jenkins |  2 +-
 2 files changed, 18 insertions(+), 1 deletion(-)
 create mode 100644 SelfServiceBuild/Build/jenkins/plf-build--parse-rules.txt

diff --git a/SelfServiceBuild/Build/jenkins/plf-build--parse-rules.txt b/SelfServiceBuild/Build/jenkins/plf-build--parse-rules.txt
new file mode 100644
index 0000000..e5a9221
--- /dev/null
+++ b/SelfServiceBuild/Build/jenkins/plf-build--parse-rules.txt
@@ -0,0 +1,17 @@
+# Log-output parsing rules for building the PLF with Jenkins.
+# (See https://plugins.jenkins.io/log-parser/ for more information).
+#
+# Needed for letting the build securely fail on build-errors because the error-level of the build-script is not <> 0 if build-errors occur.
+#
+# Currently, these rules are used by the build here:
+# https://automate.materna.de/jenkins/job/IPS-Mobility/job/MIPS-Cuss-Platform/job/TRUNK/job/01_1_MIPS_CUSS_PLATFORM/
+# (see Configuration, "Post build actions/Use project rule/...").
+#
+# 2020-03-22 mbrinkma
+# - creation
+#
+error /(?i)error .* The system cannot find the file/
+error /(?i)returned error [1-9]/
+error /BUILD FAILED/
+error /xmk.sh returned error [1-9]/
+error /: error: /
diff --git a/SelfServiceBuild/Build/release/makefile-jenkins b/SelfServiceBuild/Build/release/makefile-jenkins
index 9705505..2215393 100644
--- a/SelfServiceBuild/Build/release/makefile-jenkins
+++ b/SelfServiceBuild/Build/release/makefile-jenkins
@@ -155,7 +155,7 @@ nb_application:     build_cpp sendMailZip
 
 nb_Platform:        clean init checkout set_props build_cpp sendMailZip
 nb_Platform_java:   clean init checkout set_props build_java sendMailZip
-nb_Platform_full_nomail:   clean init set_props syncBuildResources build_cpp build_java build_maven build_gradle build_installer kidlist idllist
+nb_Platform_full_nomail:   clean init set_props syncBuildResources build_cpp build_maven build_java build_gradle build_installer kidlist idllist
 nb_Platform_full:   nb_Platform_full_nomail sendMailZip
 
 build_mavenQBG:     $(SELF)
-- 
2.41.0.windows.1

